eng/common/scripts/update-docs-ci.ps1

```diff
@@ -0,0 +1,275 @@
+# Note, due to how `Expand-Archive` is leveraged in this script,
+# powershell core is a requirement for successful execution.
```

Comment:
You can be explicit about the version needed https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_requires?view=powershell-7#-version-nn

Reply:
I'll add this soon.

Reply:
Added.